unstructured/ingest/connector/onedrive.py

```diff
@@ -22,12 +22,12 @@
 
 @dataclass
 class SimpleOneDriveConfig(BaseConnectorConfig):
-    client_id: str
-    client_credential: str = field(repr=False)
+    client_id: Optional[str]
+    client_credential: Optional[str] = field(repr=False)
```

Comment:
I believe these are both required. took a second but I think the post_init message below this needs to be updated (note that they are AND'd) I believe the ValueError message should be "Please provide all of the following...". Also, we need to bump the flag names in that error message.